src/Microsoft.Identity.Client/OAuth2/MsalTokenResponse.cs

```diff
@@ -94,5 +98,40 @@ public long ExtendedExpiresIn
 
         public DateTimeOffset AccessTokenExpiresOn { get; private set; }
         public DateTimeOffset AccessTokenExtendedExpiresOn { get; private set; }
+
+        public string Authority { get; private set; }
```

Comment:
Isn't this already available on the OAuth2ResponseBase ? Assuming it should be specified as a DataMember ?

Reply:
we just have error, suberror, errorDescription, errorCodes, CorrelationId, and Claims in the OAuth2ResponseBase

---
In reply to: [258727835](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/881#discussion_r258727835) [](ancestors = 258727835)